engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -119,9 +121,10 @@ namespace dmGameSystem
             dmPhysics::HWorld2D m_World2D;
             dmPhysics::HWorld3D m_World3D;
         };
-        float m_LastDT; // Used to calculate joint reaction force and torque.
-        uint8_t m_ComponentIndex;
-        uint8_t m_3D : 1;
+        float       m_CurrentDT;    // Used to calculate joint reaction force and torque.
```

Comment:
Since the variable was always set _before_ the physics update, it seemed better to name it _current_ dt.